js/main.js

```diff
@@ -0,0 +1,37 @@
+var $form = document.querySelector('form');
+var $placeholderImg = document.querySelector('.placeholder-img');
+var $photoURL = document.querySelector('#photo-url');
+var $title = document.querySelector('#title');
+var $notes = document.querySelector('#notes');
+
+$photoURL.addEventListener('input', handleInput);
+
+function handleInput(event) {
+  var enteredURL = event.target.value;
+  $placeholderImg.setAttribute('src', enteredURL);
+}
+
+$form.addEventListener('submit', handleSubmit);
+
+var newEntry = {};
```

Comment:
You should keep this inside the `handleSubmit` function, it doesn't need to be global.

Reply:
got it